api/app/Notify/Client.php

```diff
@@ -417,15 +417,4 @@ private function post(string $endpoint, array $payload, array $headers = []): Re
         return Http::withHeaders($this->buildHeaders($headers))
             ->post(self::BASE_URL.$endpoint, $payload);
     }
-
-    /**
-     * GET
-     *
-     * Make a GET request to the GC Notify API
```

Comment:
I guess this is not something we are going to use in the future?

Reply:
Yeah, I originally added it since it was easy to add but I don't think we will ever use a get request with GCNotify. If we plan on using it, it will be super easy to add back in.